server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/package.scala

```diff
@@ -10,4 +12,23 @@ package object internal {
     def toHeaderSeq: List[Header] =
       underlying.asScala.map(e => Header(e.getKey, e.getValue)).toList
   }
+
+  implicit class RichChannelFuture(val cf: ChannelFuture) {
+    def close(): Unit = {
+      val _ = cf.addListener(ChannelFutureListener.CLOSE)
+    }
+  }
+
+  implicit class RichHttpMessage(val m: HttpMessage) {
+    def setHeadersFrom(response: ServerResponse[_], serverHeader: Option[String]): Unit = {
+      serverHeader.foreach(m.headers().set(HttpHeaderNames.SERVER, _))
+      response.headers
+        .groupBy(_.name)
+        .foreach { case (k, v) =>
+          m.headers().set(k, v.map(_.value).asJava)
+        }
+    }
+  }
+  val ServerCodecHandlerName = "serverCodecHandler"
+  val WebSocketControlFrameHandlerName = "wsControlFrameHandler"
```

Comment:
Cosmetic: my convention is always to declare constants as `final val`s. This generates a slightly leaner bytecode and allows the compiler to do constant folding. You can also use these constants in Java annotations.